### DISCUSSION TITLE
I'm new to mrst. Please teach me.

### DISCUSSION THREAD

**Discussion by kaede297215155** [SINTEF-AppliedCompSci/MRST#7] [Q&A]
In the 'exampleVEJohansen' included in co2lab-ve, the data of the Johansen Formation is downloaded and used for calculations. However, I would like to create my own grid and conduct studies. How should I create the grid?

**Comment by moyner** [comment]
Hi! MRST contains a large number of examples and tutorials. Writing `mrstExamples()` will show you the "core" set that cover grid construction and other core concepts. You can also write `mrstExamples ad-blackoil` to look at the examples for the ad-blackoil module. You can also have a look at the [webpage](https://www.sintef.no/projectweb/mrst/documentation/) where there are links to two free books, video tutorials and other publications.

**Reply by kaede297215155** [reply]
Hello,

Thank you for explaining everything in detail!

I will look into the ad-blackoil module.
If I come across anything I don't understand, I will ask you again.

Thank you very much!

**Reply by David16200** [reply]
Alright hello,,, please how did you install it.
